client/my-sites/customer-home/main.jsx

```diff
@@ -420,11 +524,17 @@ class Home extends Component {
 }
 
 const connectHome = connect(
-	state => {
+	( state, props ) => {
```

Comment:
Since we aren't doing any complex things with the `ownProps` argument, we can just destructure assign `displayMode`
```suggestion
	( state, { displayMode } ) => {
```

Reply:
Thank you! Updated this to simplify it. :)